ipld/kamt/src/bitfield.rs

```diff
@@ -0,0 +1,212 @@
+// Copyright 2019-2022 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use std::u64;
+
+use byteorder::{BigEndian, ByteOrder};
+use fvm_ipld_encoding::de::{Deserialize, Deserializer};
+use fvm_ipld_encoding::ser::{Serialize, Serializer};
+use fvm_ipld_encoding::strict_bytes;
+
+#[derive(Debug, Clone, PartialEq, Eq, Copy)]
+pub struct Bitfield([u64; 4]);
```

Comment:
Is this copied from somewhere?

Reply:
I mean, is it different from the hamt bitfield? If so, can we add some comments as to how? If not, can't we reuse the original?

Reply:
This question is one of the TODO items I left there. I wanted to hear what @Stebalien wants in this case because the HAMT bitfield is private to the crate. Do we want to make it public and couple these two?

Reply:
https://github.com/filecoin-project/ref-fvm/pull/1094/files#diff-6758c2ada424248b816b085b04035a3ee8fe7903fa44e237e11b26f99d6d9d4aR89 is new

Reply:
This is easier to review in https://github.com/filecoin-project/ref-fvm/pull/930/files#diff-1b21066d1c876de01d801288f2064722e144b24ba2b715736307fea4362855e4